src/DisplayEditableData.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+function display(object $card)
+{
+    return
+        "<form class='editForm' action='EditDataFunction.php?id={$card->id}' method='POST'>
+        <p>Player Details</p>
+        <p>New Player Details</p>
+        <p>{$card->PlayerName}</p>
+        <input type='text' value='{$card->PlayerName}' minlength='5' maxlength='100' name='newName'/>
+        <p>{$card->Club}</p>
+        <input type='text' value='{$card->Club}' minlength='5' maxlength='100' name='newClub'/>
+        <p>{$card->PositionName}</p>
+        <select name='Position' id='Position'>
+            <option value='1' " . ($card->PositionName == 'Goalkeeper' ? 'selected' : '') . ">Goalkeeper</option>
+            <option value='2' " . ($card->PositionName == 'Defender' ? 'selected' : '') . ">Defender</option>
+            <option value='3' " . ($card->PositionName == 'Midfielder' ? 'selected' : '') . ">Midfielder</option>
+            <option value='4' " . ($card->PositionName == 'Forward' ? 'selected' : '') . ">Forward</option>
+        </select>
```

Comment:
Rather than hard coding them, you should get the positions out of the database. At the moment, to add a new position, or change one, would require changes in the DB and in this code. 

Fair enough they are unlikely to suddenly decide to add a new position, but it's still worth thinking about